App/Screens/About/About.tsx

```diff
@@ -202,27 +192,29 @@ export function About(props: AboutProps): React.ReactElement {
 				style={styles.section}
 			>
 				<Text style={styles.h2}>
-					{t('about_beta_inaccurate_title')}
+					{t('inaccurated_beta.title')}
 				</Text>
 				<Text style={theme.text}>
-					{t('about_beta_inaccurate_message')}
+					{t('inaccurated_beta.message')}
 				</Text>
 			</ScrollIntoView>
 
 			<View style={styles.section}>
 				<Text style={styles.h2}>
-					{t('about_where_does_data_come_from_title')}
+					{t('where_does_data_come_from.title')}
 				</Text>
 				<Text style={theme.text}>
-					{t('about_where_does_data_come_from_message_1')}
-					<Text onPress={handleOpenWaqi} style={theme.link}>
-						{t('about_where_does_data_come_from_link_1')}
-					</Text>
-					{t('about_where_does_data_come_from_message_2')}
-					<Text onPress={handleOpenOpenAQ} style={theme.link}>
-						{t('about_where_does_data_come_from_link_2')}
-					</Text>
-					{t('about_where_does_data_come_from_message_3')}
+					<Trans i18nKey='where_does_data_come_from.message' t={t}>
+						{t('about_where_does_data_come_from_message_1')}
+						<Text onPress={handleOpenWaqi} style={theme.link}>
+							{t('about_where_does_data_come_from_link_1')}
+						</Text>
+						{t('about_where_does_data_come_from_message_2')}
+						<Text onPress={handleOpenOpenAQ} style={theme.link}>
+							{t('about_where_does_data_come_from_link_2')}
+						</Text>
+						{t('about_where_does_data_come_from_message_3')}
+					</Trans>
```

Comment:
Here, we could remove the _1, _2, _3, and use Trans like below L239, right?

Reply:
Obviously, I haven't finished those yet. Some new tasks coming need to handle it. (Definitely change it)